pkg/asset/installconfig/platform.go

```diff
@@ -62,10 +85,21 @@ func (a *Platform) Name() string {
 }
 
 func (a *Platform) queryUserForPlatform() (string, error) {
+	sort.Strings(validPlatforms)
 	prompt := asset.UserProvided{
-		Prompt: &survey.Select{
-			Message: "Platform",
-			Options: validPlatforms,
+		Question: &survey.Question{
+			Prompt: &survey.Select{
+				Message: "Platform",
+				Options: validPlatforms,
+			},
+			Validate: survey.ComposeValidators(survey.Required, func(ans interface{}) error {
+				choice := ans.(string)
```

Comment:
Put a check here? Just in case the dynamic type cast fails.

Reply:
No, this is correct. Programs should panic if there is a programmer error (vs an input error).